assets/queries/crossplane/azure/redis_cache_allows_non_ssl_connections/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "6c7cfec3-c686-4ed2-bf58-a1ec054b63fc",
+  "queryName": "Redis Cache Allows Non SSL Connections",
+  "severity": "MEDIUM",
+  "category": "Encryption",
+  "descriptionText": "Check if any Redis Cache resource allows non-SSL connections.",
```

Improve the description in the relates queries, please

```suggestion
"descriptionText": "Redis Cache resource should not allow non-SSL connections.",
```